README.md

```diff
@@ -1,4 +1,4 @@
-[![@stencil/helmet](https://img.shields.io/npm/v/@stencil/helmet.svg)](https://npm.im/@stencil/helmet)
+[![@stencil-community/helmet](https://img.shields.io/npm/v/@stencil-community/helmet.svg)](https://npm.im/@stencil-community/helmet)
```

Comment:
Since we've never published this package under this scope, the badge will fail/look like it's missing until we perform the publish